ROADMAP.md

```diff
@@ -179,4 +179,9 @@ connection, credential
 - [CLI] Colorizing output
 - [CLI] Mesheryctl Code coverage goal: 40%
 
+**Catalog**
+
+- [Catalog] Support for selling published designs.
```

Comment:
“Intellectual property protections for user-produced content”

please change to this. Meshery will indirectly supporting the ability to sell content by offering protections, but saying that misery will support the selling of content is inaccurate.